apps/elasticsearch/templates/services/app/configs/readiness-probe.sh.j2

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+ES_HOST="127.0.0.1"
+ES_PORT="{{ elasticsearch_api_port }}"
+ES_STARTED="/tmp/es_started"
```

Comment:
Maybe we should systematically use the `declare` command to explicitly set variables with a type. WDYT?

Reply:
It might be a good practice, yes.

We'll still have to deal with external environment variables without any type, but at least we'll be more strict on our own variables.

I'm in !